### ISSUE TITLE
Unable to save a copy&pasted in the text editor C++ logs 

### ISSUE THREAD

**Issue by H-G-Hristov** [microsoft/vscode#161085]
Type: <b>Bug</b>

1. File -> New File/New Text File
2. Copy & Paste the logs from C/C++ extension
3. Try to save them via menu, etc. 

The save dialog on macOS Monterey does not appear and I am unable to save the file.

VS Code version: Code 1.71.2 (Universal) (74b1f979648cc44d385a2286793c226e611f59e7, 2022-09-14T21:05:37.721Z)
OS version: Darwin arm64 21.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|1, 1, 2|
|Memory (System)|16.00GB (1.21GB free)|
|Process Argv|--crash-reporter-id a7eaff19-94ae-4653-b917-d9aaab19da5d|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.3.1
numbered-bookmarks|ale|8.3.1
markdown-mermaid|bie|1.15.2
mermaid-markdown-syntax-highlighting|bpr|1.4.1
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.48.1
gitlens|eam|12.2.2
EditorConfig|Edi|0.16.4
codespaces|Git|1.10.4
vscode-pull-request-github|Git|0.50.0
plantuml|jeb|2.17.4
better-cpp-syntax|jef|1.15.19
vscode-catch2-test-adapter|mat|4.3.0
git-graph|mhu|1.30.0
pylint|ms-|2022.4.0
python|ms-|2022.14.0
vscode-pylance|ms-|2022.9.20
jupyter-renderers|ms-|1.0.9
remote-containers|ms-|0.251.0
remote-ssh-edit|ms-|0.80.0
remote-wsl|ms-|0.66.3
cmake-tools|ms-|1.12.27
cpptools|ms-|1.12.4
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.9.8
swift-lang|ssw|0.8.1
shellcheck|tim|0.22.0
vscode-mermaid-editor|tom|0.15.0
cmake|twx|0.0.17
vscode-lldb|vad|1.7.4
vscode-apple-swift-format|vkn|1.3.2
vscode-swiftlint|vkn|1.7.1

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411cf:30557515
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
hb751961:30553087
dsvsc012cf:30540253
azure-dev_surveyone:30548225
i497e931:30553904
vsccc:30566497
fc301958:30567733

```

</details>

**Comment by H-G-Hristov** [comment]
I saved the logs with nano:
[vscode.failed.to.save.log](https://github.com/microsoft/vscode/files/9583119/vscode.failed.to.save.log)

**Comment by H-G-Hristov** [comment]
More information:
1. I am not able to close those Text editor tabs too even when I created a new one but I was able to close already opened C++ code editor tabs.
2. I tried recording a profile with the JavaScript profiler but when I tried to save it VSCode stopped responding.

![Screenshot 2022-09-16 at 13 24 33](https://user-images.githubusercontent.com/47526411/190618491-4b47f089-8dd6-4c82-8d44-7283307cac8c.png)
